test/migration/setup/setup_migration_test.go

```diff
@@ -9,6 +9,7 @@ import (
 
 func TestSetupMigration(t *testing.T) {
 	// given
+	t.Setenv("skip-mutating-webhook-check-on-setup", "true")
```

Comment:
I guess it's just until we merge this PR the very firs time? Then we can delete it since the "old" deployments will already have the webhook in place?
Worth adding a more detailed explanation.

Reply:
Yes, exactly.

Reply:
Added a comment to better explain it